app/views/categories/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="categories_sect">
+<div class="navigation">
+  <span class="cat_hamburger">&#9776;</span>
+   <h1>Categories</h1>
+  <%= link_to "Sign Out", destroy_user_session_path, method: :delete, class: "sign-out-butt" %>
+</div>   
+    <div class="cat_page">
+  <div id="categories">
+    <% if @categories.length == 0 %>
+      <p class="no-categories"> There are no categories yet,,,,!create one</p>
+    <% else %>
+      <% @categories.each do |category| %>
+      <div class="cat_name">
+         <%= link_to category_records_path(category) do %>
+         <%= render category %>
+        <% end %>
+      </div>
+      <% end %>
+    <% end %>
+  </div>
+  <div class="cat_link_cont">
+  <div class= "inner">
+    <%= link_to "New category", new_category_path, class:"cat_link" %>
```

Comment:
- [ Required ] I think it would be better if the styling of this button matches that of the Behance design as suggested by the previous reviewer. The button design currently varies from the one in the Behance design in the following ways.
   
   - The background color for the button in your design is `#2cc718` while the background color in the Behance design is `#5fb523`. This is evident when you look at the color palette below.
   
   - The button in your design has a `black` border around it while the button in the Behance design has none.

I would like to suggest that you refactor your code as shown below so that your design conforms to the design on Behance.

## Color palette
![Screenshot from 2023-08-19 23-04-17](https://github.com/Christelle-12/Budget-manager/assets/16392046/269f7aaa-a946-4ca8-b139-ce15fb4c27c7)

## Example code 
```CSS
.cat_link_cont {
  background-color: #5fb523;
  padding: 0.6rem;
  display: flex;
  justify-content: center;
  position: absolute;
  bottom: 0;
  width: 100%;
}

.cat_link {
  text-decoration: none;
  color: #fff;
  font-size: 1.5rem;
  background-color: #5fb523;
}

.inner {
  padding: 1rem;
  border: 1px solid #fff;
  background-color: #5fb523;
  width: 100%;
  text-align: center;
  border-radius: 0.3rem;
}
```